PWGCF/Femto3D/Core/femto3dPairTask.h

```diff
@@ -253,6 +260,24 @@ bool FemtoPair<TrackType>::IsClosePair(const float& deta, const float& dphi, con
   return false;
 }
 
+template <typename TrackType>
+float FemtoPair<TrackType>::GetAvgSep() const
+{
+  if (_first == NULL || _second == NULL)
+    return -100.f;
+  if (_magfield1 * _magfield2 == 0)
+    return -100.f;
+
+  float dtheta = THETA(_first->eta()) - THETA(_second->eta());
+  float res = 0.0;
+
+  for (const auto& radius : TPCradii) {
+    res += sqrt(pow(2.0 * radius * sin(0.5 * GetPhiStarDiff(radius)), 2) + pow(2.0 * radius * sin(0.5 * dtheta), 2));
+  }
+
+  return 100.0 * res / 9;
```

Comment:
I would recommend using the length of the `TPCradii` instead of `9`

Reply:
Dear @victor-gonzalez , thank you your comment. The size if fixed so I used just 9, now I changed it to std::array and calling the size() method. Is it OK like this?

Reply:
Thanks! It is OK
My point was that if someone in the future changes the size of the array by including more radii they only need to care about the array and nothing else

Reply:
ah, I see. Thanks!